apps/render_cf.py

```diff
@@ -11,21 +11,7 @@ def snake_to_pascal_case(input_string: str):
     return ''.join([i.title() for i in split_string])
 
 
-def render_template(template_file, job_types, env):
-    output_file = template_file.with_suffix('')
-
-    template = env.get_template(str(template_file))
-    with open(output_file, 'w') as f:
-        f.write(template.render(job_types=job_types, json=json, snake_to_pascal_case=snake_to_pascal_case))
-
-
-def render_templates(job_types):
-    env = get_env()
-    for template_file in Path('.').glob('**/*.j2'):
-        render_template(PurePosixPath(template_file), job_types, env)
-
-
-def get_env():
```

Comment:
I found a lot of this unnecessary and the logic easier to follow with just a `render_templates` fn. I can back out some or all of it though.